### PR TITLE
Fix docstring for heatmaps_to_keypoints

### LINKED ISSUE #7148: docstring for `heatmaps_to_keypoints` does not match return values.
The docstring for the function `heatmaps_to_keypoints` in `torchvision/models/detection/roi_heads.py` does not match the return values.
```python
def heatmaps_to_keypoints(maps, rois):
    """Extract predicted keypoint locations from heatmaps. Output has shape
    (#rois, 4, #keypoints) with the 4 rows corresponding to (x, y, logit, prob)
    for each keypoint.
    """
...
return xy_preds.permute(0, 2, 1), end_scores
```
The function returns 2 tensors.  The first tensor has the shape `(#rois, #keypoints, 3)` with the last dimension corresponding to `(x, y, vis)`.  The second tensor has shape `(#rois, #keypoints)` the values of which correspond to... logits?  Some explanation on what these `end_scores` are used for and/or how to use them would also be helpful.

### PR DESCRIPTION
**Pull request by raimbekovm**
## Summary

Fixes #7148

The docstring for `heatmaps_to_keypoints` was incorrect:
- Stated output shape as `(#rois, 4, #keypoints)` but actual shape is `(#rois, #keypoints, 3)`
- Stated last dimension contains `(x, y, logit, prob)` but actually contains `(x, y, visibility)`
- Did not mention that the function returns a tuple of two tensors

## Changes

Updated docstring to follow Google style with accurate `Args` and `Returns` sections that match the actual function behavior:
- `xy_preds`: `Tensor[K, N, 3]` containing `(x, y, v)` coordinates and visibility
- `scores`: `Tensor[K, N]` containing heatmap scores at predicted locations

### CODE REVIEW

**Comment by raimbekovm** [comment]
Thank you for the review!
